### DISCUSSION TITLE
Server Components with Sass modules

### DISCUSSION THREAD

**Discussion by dmitriigirskii** [vercel/next.js#31453] [App Router]
Hi in **^12.0.4-canary.10** can't modular sass be used?

`import styles from "./index.module.sass"`
`<main className={styles.index}>Index page</main>`

After reload the **main** have class, but class not have styles

How do you do it right?

**Comment by IshanKBG** [comment]
Currently css modules doesn't work with react server components
You can checkout what's not supported with react server components in nextjs [here](https://nextjs.org/docs/advanced-features/react-18#unsupported-nextjs-apis)

**Comment by sviridoff** [comment]
Are there any plan to make it support? I seen recently css modules support was added
